static/js/controllers/contacts-content.js

```diff
@@ -87,7 +87,7 @@ angular.module('inboxControllers').controller('ContactsContentCtrl',
           getTasks();
         })
         .catch(function(err) {
-          if (err.error === 'not_found') {
```

Comment:
What was going on here?  Was this condition never fulfilled previously?

Reply:
Yeah I think it worked once, and then stopped getting used when the linked contacts change came in.